.github/docker/linux_amd64/Dockerfile

```
@@ -0,0 +1,126 @@
+
+# Install GIT
+RUN wget https://github.com/git/git/archive/refs/tags/v2.25.0.tar.gz && \
+    tar xvf v2.25.0.tar.gz && \
+    cd git-2.25.0 && \
+    make && \
+    make prefix=/usr install
+
+# Setup VCPKG n a mounted volume TODO: figure out how to cache this
+ARG vcpkg_url
+ARG vcpkg_commit
+RUN mkdir /vcpkg && \
```

Hadolint (2.12.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)